code/script.js

```diff
@@ -1,19 +1,138 @@
-// Start here
+// STEP 1 - Welcome and introduction//
+//Alertmessage: Welcome
+//Promptmessage: Please enter name
+//Alertmessage: Thank you + name
+alert(
+  `Welcome to Mirela´s Pizzeria. Ready to start your order? - Click 'OK' to begin.`
+)
+
+let text;
+let NAME = prompt("Please enter your full name");
```

Comment:
I think a variable should be in lowercase. You can use it, but it might be confusing. 
Here is a link to an article , in which people discussed it. It helped me to understand better how to use lowercase and uppercase in declaring variables. 

https://teamtreehouse.com/community/does-the-first-letter-of-a-variable-name-have-to-be-lowercase#:~:text=In%20JavaScript%2C%20you%20can%20technically,constants%2C%20methods%2C%20and%20classes.